### ISSUE TITLE
image cannot sync across device

### ISSUE THREAD

**Issue by LotuSrc** [AppFlowy-IO/AppFlowy-Cloud#196]
I wrote a simple markdown with some words and an image. And on another laptop, appflowy can only display the text correctly. The image cannot be displayed.
![image](https://github.com/AppFlowy-IO/AppFlowy-Cloud/assets/24475165/db47ad56-c656-476d-95f7-cf3a26eaa2ac)

**Comment by speed2exe** [comment]
@LotuSrc Thank you for the feedback. This is a known issue and will be fixed in the future

**Comment by poldueta** [comment]
Faced with the same issue. Also realised that images which were pasted from clipboard are not synced. But if image inserted via `/image` command or via `+` menu interface it will be synced correctly.

**Comment by speed2exe** [comment]
@poldueta @LotuSrc 
with the latest version of appflowy cloud and the app, this should be fixed. if you still face this issue, do let me know the steps to reproduce

**Comment by poldueta** [comment]
Checked just now - still the same behaviour.

Default docker compose with the `appflowyinc/appflowy_cloud:latest` image

Login via Google to the desktop app v0.4.8 (macOS), it mostly fails to login (looks like something is broken here), but if it finally will, then create empty page and paste screenshot from clipboard. 

Let's check minio storage - there is no new file was uploaded. 

Then trying to add another image after that screenshot, now via `/image` command. 

Check minio storage again - one new image was uploaded.

**Comment by poldueta** [comment]
Login problems were fixed in 0.4.9 

Unfortunately, uploading to S3 for images which were pasted to document from clipboard still doesn't work. No new new objects appears. 

Adding images via `/image` command works correctly, uploading works.

**Comment by speed2exe** [comment]
maybe you can check the logs in the appflowy-cloud service while you were doing the upload @poldueta

**Comment by poldueta** [comment]
Checked and there is nothing to see in the appflowy-cloud service logs. Literally. 

When I tried to add images via `/image` command and works correctly, uploading works and logs displayed request which comes to backend.

When I paste image to document from clipboard - nothing to see on a backend part. Because client app doesn't do an upload.

Looks like it's a client related issue, both mobile and desktop.
